src/App.tsx

```diff
@@ -11,58 +11,63 @@ import PrimeCheckComponent from './components/organisms/math-functions/prime-che
 import FactorialComponent from './components/organisms/math-functions/factorial/factorial.component';
 import VowelCountComponent from './components/organisms/math-functions/vovel-count/vowel-count.component';
 import FibonacciComponent from './components/organisms/math-functions/fibonacci/fibonacci.component';
-
 import ApiDataFetcher from './components/molecules/other-functions/api-data-fetcher/api-data-fetcher.component';
 import CsvLoader from './components/molecules/other-functions/csv-loader/csv-loader.component';
 import DelayFunction from './components/molecules/other-functions/delay/delay.component';
+import { CssBaseline, GlobalStyles, Grid } from '@mui/material';
 
-import './App.scss';
+const globalStyles = `
+  body {
+    font-family: 'Roboto', sans-serif;
+    background-color: #366a4b;
+  }
+`;
 
 const App: React.FC = () => {
   return (
     <BrowserRouter>
-      <div className='wrapper'>
+      <CssBaseline />
+      <GlobalStyles styles={globalStyles} />
+      <Grid container direction="column" justifyContent="center" alignItems="center" style={{ minHeight: '100vh' }}
```

Comment:
```suggestion
      <Grid container direction="column" justifyContent="center" alignItems="center" sx={{ minHeight: '100vh' }}
```
chyba tak powinno  być - mam rację?